tests/dialects/test_postgres.py

```diff
@@ -608,7 +608,13 @@ def test_postgres(self):
                 "postgres": "x / y ^ z",
             },
         )
-
+        self.validate_all(
+            "SELECT pgc.relname::CHARACTER VARYING AS textin FROM pg_class pgc WHERE pgc.relname <> 'my_table'::name",
```

Comment:
We don't really care about the `CHARACTER VARYING` bit in this test, we test this elsewhere. I think it's worth changing that with `VARCHAR` so you can also add the redshift -> postgres direction here using the `read` kwarg.